### ISSUE TITLE
 3.10.0-1127.el7.x86_64(CR and future 7.8) and EL7.7  zfs: disagrees about version of symbol posix_acl_valid

### ISSUE THREAD

**Issue by truatpasteurdotfr** [openzfs/zfs#10208]
### System information

Type | Version/Name
 --- | --- 
Distribution Name	| CentOS
Distribution Version	| 7.7+CR (would become 7.8)
Linux Kernel	| 3.10.0-1127.el7.x86_64
Architecture	| x86_64
ZFS Version	| 0.8.3-1
SPL Version	| 0.8.3-1

### Describe the problem you're observing
kmod-zfs for 7.7 is not running on the kernel for 7.8 series

### Describe how to reproduce the problem
enable the cr repository and install kernel-3.10.0-1127.el7.x86_64
zfs kmod would not load
```
Apr 15 13:57:28 nestor kernel: zfs: disagrees about version of symbol posix_acl_valid
Apr 15 13:57:28 nestor kernel: zfs: Unknown symbol posix_acl_valid (err -22)

```
Installing kernel-devel-3.10.0-1127.el7.x86_64 and
rebuilding the kmod-zfs  for 3.10.0-1127.el7.x86_64 yields a loadable zfs module
while waiting to EL7.8 from openzfs.

**Comment by sandhoners123** [comment]
Are you sure that kmod-zfs was built for your kernel version. Linux does not have a stable kabi

**Comment by truatpasteurdotfr** [comment]
@sandhoners123 yes, see also https://github.com/openzfs/zfs/wiki/RHEL-and-CentOS
I expect a EL7.8 Package soon, my purpose is  a word of caution for early testers.

**Comment by chriss745** [comment]
> What about zfs 0.7.x? Does the kmod version load/install on the lastest CentOS 7.7 CR kernel? Thanks.

I have the same problem. I tried with zfs 0.7.x with kernel 3.10.0-1127, unfortunately that neither works:

[root@centos zfs]# dmesg | tail
[ 1382.723943] zfs: disagrees about version of symbol fnvlist_lookup_nvpair
[ 1382.728789] zfs: Unknown symbol fnvlist_lookup_nvpair (err -22)
[ 1382.733373] zfs: disagrees about version of symbol nvpair_value_string
[ 1382.738414] zfs: Unknown symbol nvpair_value_string (err -22)
[ 1382.742848] zfs: disagrees about version of symbol __kstat_install
[ 1382.748309] zfs: Unknown symbol __kstat_install (err -22)
[ 1382.752450] zfs: disagrees about version of symbol zfs_prop_get_type
[ 1382.756973] zfs: Unknown symbol zfs_prop_get_type (err -22)
[ 1382.760821] zfs: disagrees about version of symbol vn_fsync
[ 1382.764729] zfs: Unknown symbol vn_fsync (err -22)
[root@centos zfs]# modinfo zfs
filename:       /lib/modules/3.10.0-1127.el7.x86_64/weak-updates/zfs/zfs/zfs.ko
version:        0.7.13-1
license:        CDDL
author:         OpenZFS on Linux
description:    ZFS
alias:          devname:zfs
alias:          char-major-10-249
retpoline:      Y
rhelversion:    7.6
srcversion:     CD1489EAB5842683DC3584F
depends:        spl,znvpair,zcommon,zunicode,zavl,icp
vermagic:       3.10.0-957.5.1.el7.x86_64 SMP mod_unload modversions

**Comment by hoonetorg** [comment]
CentOS 7.8 is released now.
https://lists.centos.org/pipermail/centos-announce/2020-April/035696.html
https://distrowatch.com/10894

Maybe it's time to provide http://download.zfsonlinux.org/epel/zfs-release.el7_8.noarch.rpm and fill a repo located at http://download.zfsonlinux.org/epel/7.8 with packages ;) .

**Comment by hoonetorg** [comment]
#10261
